app/lib/pre_assembly/batch.rb

```diff
@@ -173,32 +169,12 @@ def discover_containers_via_manifest
       manifest_rows.map { |r| bundle_dir_with_path r[:object] }
     end
 
-    # A method to discover object containers or stageable items.
-    # Takes a root path (e.g, bundle_dir) and a discovery data structure.
-    # The latter drives the two-stage discovery process:
-    #   - A glob pattern to obtain a list of dirs and/or files.
-    #   - A regex to filter that list.
-    # FIXME: configuration of aforementioned stageable_discovery data structure is currently unsupported.
-    # the remnants are vestiges of v3-legacy branch that we didn't have time to properly disposition.
-    # behavior should either be fully removed, or properly reimplemented and tested.
-    # see: #274, https://github.com/sul-dlss/pre-assembly/blob/v3-legacy/config/projects/TEMPLATE.yaml
-    # rubocop:disable Metrics/AbcSize
-    # rubocop:disable Metrics/CyclomaticComplexity
+    # A method to discover stageable items (i.e. files) with a given object folder.
+    # Takes a root path of the object folder (as supplied in the manifest).
```

Comment:
same here.  Maybe we can have better distinguishing names in our code from "manifest" and "file_manifest" ?   Or maybe this isn't confusing to anyone else.